### ISSUE TITLE
王老师，请问有没有Alexnet 的default yaml参数可以达到Resnet的结果？

### ISSUE THREAD

**Issue by webzerg** [jindongwang/transferlearning#248]
大家好， 您好！ 现在把yaml里的Resnet 换成Alexnet，accuracy只有20% 多，请问有没有谁试过用Alexnet 需要怎么设置参数可以达到Resnet类似的结果？ 谢谢

**Comment by houwenxin** [comment]
你好，可以贴出你的使用的DA方法还有log吗？

**Comment by webzerg** [comment]
@houwenxin 这是我 允许的命令，我只把DAN.yaml里面的resnet50换成了alexnet。

 % python3 main.py --config DAN/DAN.yaml --data_dir /Users/xin/causal/code/transferlearning/code/DeepDA/datasets/office31 --src_domain dslr --tgt_domain amazon | tee DAN_D2A.log

Namespace(config='DAN/DAN.yaml', seed=1, num_workers=3, backbone='alexnet', use_bottleneck=True, data_dir='/Users/nizhao/xin/causal/code/transferlearning/code/DeepDA/datasets/office31', src_domain='dslr', tgt_domain='amazon', batch_size=32, n_epoch=20, early_stop=0, epoch_based_training=False, n_iter_per_epoch=500, lr=0.003, momentum=0.9, weight_decay=0.0005, lr_gamma=0.0003, lr_decay=0.75, lr_scheduler=True, transfer_loss_weight=0.5, transfer_loss='mmd', device=device(type='cpu'))

**Comment by houwenxin** [comment]
可以提供训练过程中loss，acc等的变化吗？DAN_D2A.log里应该有

**Comment by webzerg** [comment]
嗯， 我在重新运行等结果，电脑比较慢。结果出来了马上给你。

**Comment by webzerg** [comment]
@houwenxin 可不可以麻烦你找一个DAN或DSAN，在在yaml里把resnet50换成alexnet，看一看你那边运行的结果怎么样？感谢🙏

**Comment by webzerg** [comment]
我之前实验的时候对各种模型（DAN或DSAN），alexnet 结果比resnet50的结果都差了20%-30%左右。

**Comment by houwenxin** [comment]
不好意思，我现在暂时没有条件复现实验，你可以先调整一下学习率吗？可以试一下大一点的学习率。log也可以发上来看一下，我看一下可能的原因

**Comment by webzerg** [comment]
@houwenxin 我试过不同的学习率了，但感觉差别都不大。 下面是log。谢谢！

Namespace(config='DAN/DAN.yaml', seed=1, num_workers=3, backbone='alexnet', use_bottleneck=True, data_dir='/Users/nizhao/xin/causal/code/transferlearning/code/DeepDA/datasets/office31', src_domain='dslr', tgt_domain='amazon', batch_size=32, n_epoch=20, early_stop=0, epoch_based_training=False, n_iter_per_epoch=500, lr=0.003, momentum=0.9, weight_decay=0.0005, lr_gamma=0.0003, lr_decay=0.75, lr_scheduler=True, transfer_loss_weight=0.5, transfer_loss='mmd', device=device(type='cpu'))
Epoch: [ 1/20], cls_loss: 0.1643, transfer_loss: 0.2074, total_Loss: 0.2680, test_loss 2.620415, test_acc: 40.3266
Epoch: [ 2/20], cls_loss: 0.0136, transfer_loss: 0.1974, total_Loss: 0.1123, test_loss 2.563804, test_acc: 40.1491
Epoch: [ 3/20], cls_loss: 0.0123, transfer_loss: 0.1959, total_Loss: 0.1103, test_loss 2.556465, test_acc: 40.2556
Epoch: [ 4/20], cls_loss: 0.0117, transfer_loss: 0.1959, total_Loss: 0.1097, test_loss 2.517622, test_acc: 39.4036
Epoch: [ 5/20], cls_loss: 0.0117, transfer_loss: 0.1944, total_Loss: 0.1089, test_loss 2.530876, test_acc: 39.4746
Epoch: [ 6/20], cls_loss: 0.0113, transfer_loss: 0.1964, total_Loss: 0.1095, test_loss 2.505117, test_acc: 40.0071
Epoch: [ 7/20], cls_loss: 0.0111, transfer_loss: 0.1954, total_Loss: 0.1088, test_loss 2.541836, test_acc: 39.9006
Epoch: [ 8/20], cls_loss: 0.0112, transfer_loss: 0.1969, total_Loss: 0.1096, test_loss 2.497745, test_acc: 39.9006
Epoch: [ 9/20], cls_loss: 0.0110, transfer_loss: 0.1951, total_Loss: 0.1086, test_loss 2.557159, test_acc: 40.1846
Epoch: [10/20], cls_loss: 0.0110, transfer_loss: 0.1968, total_Loss: 0.1095, test_loss 2.495049, test_acc: 40.3976
Epoch: [11/20], cls_loss: 0.0111, transfer_loss: 0.1955, total_Loss: 0.1088, test_loss 2.553153, test_acc: 39.9716
Epoch: [12/20], cls_loss: 0.0109, transfer_loss: 0.1944, total_Loss: 0.1081, test_loss 2.541713, test_acc: 40.6816
Epoch: [13/20], cls_loss: 0.0111, transfer_loss: 0.1977, total_Loss: 0.1099, test_loss 2.500549, test_acc: 40.4331
Epoch: [14/20], cls_loss: 0.0110, transfer_loss: 0.1951, total_Loss: 0.1085, test_loss 2.469287, test_acc: 40.5041
Epoch: [15/20], cls_loss: 0.0110, transfer_loss: 0.1981, total_Loss: 0.1100, test_loss 2.494790, test_acc: 40.2911
Epoch: [16/20], cls_loss: 0.0110, transfer_loss: 0.1956, total_Loss: 0.1088, test_loss 2.484786, test_acc: 40.6461

**Comment by jindongwang** [comment]
@webzerg 看结果是一直在涨点的，所以你可以尝试一下：调大mmd权重，或者多一些epoch来看看。我们提供的参数都是在resnet-50上调过的，所以可能不能直接在AlexNet上取得最好的效果，你可以自己调一下看看。

**Comment by webzerg** [comment]
我试过了调整很多参数，效果都不明显. 可以帮调一下吗？谢谢

**Comment by houwenxin** [comment]
我做了一下实验，最好结果是在42%左右，resnet本身是一个比较强大的backbone，在imagenet的结果比alexnet好很多，所以想要将alexnet调到和resenet相近的结果比较困难；我们还是建议你使用resnet作为backbone

**Comment by jindongwang** [comment]
@webzerg 请问你的问题解决了吗？
